### PR TITLE
[stable/spinnaker] Allow users to register arbitrary Kubernetes clusters

### PR DESCRIPTION
**Pull request by viglesiasce**
**What this PR does / why we need it**:
Adds ability to manage arbitrary clusters using the Spinnaker chart. Currently you can only (easily) manage the one that you deploy to.

### CODE REVIEW

**Comment by ameer00** [comment]
Tested on a GKE cluster running 1.9.6-gke.1.  Works great!!

File: stable/spinnaker/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Open source, multi-cloud continuous delivery platform for releasing software changes with high velocity and confidence.
 name: spinnaker
-version: 0.4.1
+version: 0.4.2
```
Comment (timstoop) [inline stable/spinnaker/Chart.yaml]: As you're adding a new feature, please bump the minor version.

**Review by timstoop — CHANGES_REQUESTED** [review]
Nice change!

**Comment by viglesiasce** [comment]
Thanks for the review @timstoop! I updated the version and fixed an issue with the default pod spec.

**Pull request merged**